explainerdashboard/explainers.py

```diff
@@ -25,8 +25,8 @@
 
 import shap
 
-import dtreeviz
-from dtreeviz.models.shadow_decision_tree import ShadowDecTree
+from dtreeviz import *
+from dtreeviz.trees import ShadowDecTree
```

Comment:
Does this work in both 1.4.1 and 2.0.0?  Maybe this is safer and more specific?

```
from dtreeviz.models.shadow_decision_tree import ShadowDecTree
```

Reply:
@haizadtarik did you have a chance to test the imports both with dtreeviz 1.4.1 and 2.0?

Reply:
Yes I tested to run the sample code from [README](https://github.com/oegedijk/explainerdashboard#examples-of-use) on dtreeviz v2.0.0 and dtreeviz v1.4.1 and did not encounter the import error and managed to bring up the dashboard ui

Using @parrt suggestion also worked on both version.

Let me know which is the best way to import the `ShadowDecTree` and I'll update this PR

Reply:
Let's go with @parrt 's suggestion then:

```
from dtreeviz.models.shadow_decision_tree import ShadowDecTree
```